autoload/lsp/hover.vim

```diff
@@ -90,11 +90,6 @@ def HoverWinFilterKey(hoverWin: number, key: string): bool
     keyHandled = true
   endif
 
-  if !keyHandled
-    # For any other key, close the hover window
-    hoverWin->popup_close()
-  endif
-
```

Comment:
Pressing Escape should close the hover popup window.  With the above change, the escape key will not close the hover window.  An explicit check for `Esc` key should be added to close the popup window.